### DISCUSSION TITLE
How subscribe change progress?

### DISCUSSION THREAD

**Discussion by DenisVASI9** [OptimalBits/bull#2192] [Q&A]
Hi, I would like to give progress about work via sse, is it possible to do something like 

```typescript
job.on('progress', (value) => observable.next(value))
```

?

It's not exactly a mistake, but I couldn't find a source explaining it. Thanks in advance.

**Comment by manast** [comment]
Well, this application specific. The progress event will be triggered by the progress event... you probably want to use the global version of the event instead like ```queue.on("global:progress", (jobId, progress) => {});```
